editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -3985,6 +3986,18 @@ void CanvasItemEditor::edit(CanvasItem *p_canvas_item) {
 	if (selection.size() != 1 || Object::cast_to<Node>(selection[0]) != p_canvas_item) {
 		_reset_drag();
 	}
+
+	// If this CanvasItem's shader is already open in the shader editor, switch to it.
+	ShaderEditorPlugin *shaderEditor = Object::cast_to<ShaderEditorPlugin>(EditorNode::get_singleton()->get_editor_data().get_editor("Shader"));
```

Comment:
We use `snake_case` for identifiers.

Reply:
Missed that one sorry, changed in both files in a new commit.